libr/core/cmd_anal.c

```diff
@@ -6727,11 +6787,11 @@ static void cmd_agraph_print(RCore *core, const char *input) {
 		r_agraph_foreach_edge (core->graph, agraph_print_edge, NULL);
 		break;
 	case 'J':
-	case 'j':
+	case 'j': {
 		r_cons_printf ("{\"nodes\":[");
```

Comment:
Why don't you use pj here too?

Reply:
I thought I'd go file wise. But now I made the changes like you said.